extensions/amp-form/0.1/amp-form.js

```diff
@@ -769,12 +773,19 @@ export class AmpForm {
   }
 
   /**
-   * Transition the form to the submit success state.
+   * Transition the form to the submit-success or submit-error state depending on the response status.
    * @param {!JsonObject} response
    * @return {!Promise}
    * @private
    */
-  handleSsrTemplateSuccess_(response) {
+  handleSsrTemplateResponse_(response) {
+    if (
+      response.init &&
+      response.init.status &&
+      response.init.status > SUBMIT_ERROR_STATUS_THRESHOLD
```

Comment:
If these are HTTP status codes, should this be `>=` 300? 

Nit: you can just inline `300` here.